diff --git a/bzip2-1.0.6/makefile.msc b/bzip2-1.0.6/makefile.msc
index 6a628a7..e0fccb0 100644
--- a/bzip2-1.0.6/makefile.msc
+++ b/bzip2-1.0.6/makefile.msc
@@ -4,7 +4,7 @@
 # Fixed up by JRS for bzip2-0.9.5d release.
 
 CC=cl
-CFLAGS= -DWIN32 -MD -Ox -D_FILE_OFFSET_BITS=64 -nologo
+CFLAGS= -DWIN32 -MDd -Ox -D_FILE_OFFSET_BITS=64 -nologo
 
 OBJS= blocksort.obj  \
       huffman.obj    \
diff --git a/libxml2-2.9.1/win32/Makefile.msvc b/libxml2-2.9.1/win32/Makefile.msvc
index 2b3d544..1894def 100644
--- a/libxml2-2.9.1/win32/Makefile.msvc
+++ b/libxml2-2.9.1/win32/Makefile.msvc
@@ -98,7 +98,7 @@ ARFLAGS = /nologo
 
 # Optimisation and debug symbols.
 !if "$(DEBUG)" == "1"
-CFLAGS = $(CFLAGS) /D "_DEBUG" /Od /Z7
+CFLAGS = $(CFLAGS) /D "_DEBUG" /Od /Z7 /MDd
 LDFLAGS = $(LDFLAGS) /DEBUG
 !else
 CFLAGS = $(CFLAGS) /D "NDEBUG" /O2 
diff --git a/proj-4.8.0/nad/makefile.vc b/proj-4.8.0/nad/makefile.vc
index 1166104..1730225 100644
--- a/proj-4.8.0/nad/makefile.vc
+++ b/proj-4.8.0/nad/makefile.vc
@@ -30,5 +30,5 @@ install-all:
 	-mkdir $(PROJ_LIB_DIR)
 	for %f in ( $(OTHER_FILES) ) do copy %f $(PROJ_LIB_DIR)
 	if exist conus $(MAKE) /f makefile.vc install-nadgrids
-	-copy *.gsb $(PROJ_LIB_DIR)
-	-copy ntv1_can.dat $(PROJ_LIB_DIR)
+#	-copy *.gsb $(PROJ_LIB_DIR)
+#	-copy ntv1_can.dat $(PROJ_LIB_DIR)
diff --git a/proj-4.8.0/nmake.opt b/proj-4.8.0/nmake.opt
index 79637ea..e6d8d82 100644
--- a/proj-4.8.0/nmake.opt
+++ b/proj-4.8.0/nmake.opt
@@ -4,7 +4,7 @@
 
 # Directory tree where PROJ will be installed.
 !IFNDEF INSTDIR
-INSTDIR=C:\PROJ
+INSTDIR=C:\\PROJ
 !ENDIF
 
 # Set the following to the directory where the PROJ distribution data files
@@ -16,21 +16,15 @@ INSTDIR=C:\PROJ
 # precedence at runtime over the setting of the PROJ_LIB_DIR macro stored in
 # the compiled software.
 !IFNDEF PROJ_LIB_DIR
-PROJ_LIB_DIR=$(INSTDIR)\SHARE
+PROJ_LIB_DIR=$(INSTDIR)\\share
 !ENDIF
 
 # Uncomment the first for an optimized build, or the second for debug.
-!IFNDEF OPTFLAGS
-!IFNDEF DEBUG
-OPTFLAGS=	/nologo /Ox /Op /MD
-!ELSE
-OPTFLAGS=	/nologo /Zi /MD /Fdproj.pdb
-!ENDIF
-!ENDIF
+OPTFLAGS=	/nologo /Zi /MDd /Fdproj.pdb
 
 # Uncomment the first for linking exes against DLL or second for static
-EXE_PROJ =	proj_i.lib
-#EXE_PROJ =	proj.lib
+#EXE_PROJ =	proj_i.lib
+EXE_PROJ =	proj.lib
 
 # Set the version number for the DLL.  Normally we leave this blank since
 # we want software that is dynamically loading the DLL to have no problem
diff --git a/proj-4.8.0/src/makefile.vc b/proj-4.8.0/src/makefile.vc
index c89ec76..47352c8 100644
--- a/proj-4.8.0/src/makefile.vc
+++ b/proj-4.8.0/src/makefile.vc
@@ -130,7 +130,7 @@ install: all
 	-mkdir $(INSTDIR)\lib
 	-mkdir $(INSTDIR)\include
 	copy *.exe $(INSTDIR)\bin
-	copy *.dll $(INSTDIR)\bin
+	copy *.pdb $(INSTDIR)\bin
 	copy *.lib $(INSTDIR)\lib
 	copy proj_api.h $(INSTDIR)\include
 	
diff --git a/zlib-1.2.8/win32/Makefile.msc b/zlib-1.2.8/win32/Makefile.msc
index 67b7731..7bc3b87 100644
--- a/zlib-1.2.8/win32/Makefile.msc
+++ b/zlib-1.2.8/win32/Makefile.msc
@@ -26,7 +26,7 @@ AS = ml
 LD = link
 AR = lib
 RC = rc
-CFLAGS  = -nologo -MD -W3 -O2 -Oy- -Zi -Fd"zlib" $(LOC)
+CFLAGS  = -nologo -MDd -W3 -O2 -Oy- -Zi -Fd"zlib" $(LOC)
 WFLAGS  = -D_CRT_SECURE_NO_DEPRECATE -D_CRT_NONSTDC_NO_DEPRECATE
 ASFLAGS = -coff -Zi $(LOC)
 LDFLAGS = -nologo -debug -incremental:no -opt:ref
